packages/block-library/src/block/edit.js

```diff
@@ -104,14 +110,18 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 		<RecursionProvider>
 			<div { ...blockProps }>
 				<BlockControls>
-					<ToolbarGroup>
-						<ToolbarButton
-							onClick={ () => convertBlockToStatic( clientId ) }
-							label={ __( 'Convert to regular blocks' ) }
-							icon={ ungroup }
-							showTooltip
-						/>
-					</ToolbarGroup>
+					{ canRemove && (
```

Comment:
I guess we can move this check one level up for now.

Reply:
Do you mean `BlockControls`?

Reply:
Yes. Sorry for not clarifying 😄